test/read-file-sync.test.js

```diff
@@ -43,7 +43,7 @@ describe('+ readFileSync()', function () {
         jf.readFileSync(file)
       }, function (err) {
         assert(err instanceof Error)
-        assert(err.message.match(file))
+        assert(err.message.match(/somefile\.json/))
```

Comment:
This test fails when run on Windows with node v0.12.x.

![image](https://cloud.githubusercontent.com/assets/946879/10730981/05c5f4d8-7beb-11e5-99c8-fe011676a152.png)